recipes/devices/families/x86.sh

```diff
@@ -246,7 +243,7 @@ EOF
   sed -i "s/%%IMGPART%%/${UUID_IMG}/g" ${grub_cfg}
   sed -i "s/%%BOOTPART%%/${UUID_BOOT}/g" ${grub_cfg}
   sed -i "s/%%DATAPART%%/${UUID_DATA}/g" ${grub_cfg}
-  sed -i "s/%%KRNL%%/${KRNL}/g" ${grub_cfg}
+
```

Comment:
Wouldn't this lead to a nonbootable image -- [`grub.cfg`](https://github.com/volumio/platform-x86/blob/0e4dd9c56ad0c6224d7854685abdf03f454b2278/packages-buster/efi/BOOT/grub.cfg#L117) will try and load `%%KRNL%%` as the kernel.

Reply:
Ah, https://github.com/volumio/platform-x86/commit/e9aad47c415c73b772e8542d63f02e1720130bae covers that! 👍

Reply:
> Wouldn't this lead to a nonbootable image -- [`grub.cfg`](https://github.com/volumio/platform-x86/blob/0e4dd9c56ad0c6224d7854685abdf03f454b2278/packages-buster/efi/BOOT/grub.cfg#L117) will try and load `%%KRNL%%` as the kernel.

Yes, but the platform files were updated, this works fine